packages/ui/src/components/va-input/components/VaInputWrapper/_variables.scss

```diff
@@ -0,0 +1,20 @@
+:root {
+  // Util CSS variables used to change component style during runtime
+  --va-input-wrapper-color: var(--va-primary);
+  --va-input-wrapper-background: var(--va-input-color);
```

Comment:
Lets create an issue and discuss about it later, for now we merge this PR.

Reply:
I believe we need to look on all outdated CSS variables. We have a lot of useless CSS variables, for example crucial display variables (if user will change it - component will be destroyed). 

We also have overridden CSS variables by JS, live `--va-time-picker-color`.  These variables must be removed from variables.scss, because the only way user can change them is to pass prop to component.